src/test/java/oracle/kubernetes/operator/create/CreateDomainInputsFileTest.java

```diff
@@ -61,6 +61,7 @@ public void defaultInputsFile_hasCorrectContents() throws Exception {
         .namespace("default")
         .weblogicDomainStorageNFSServer("")
         .weblogicDomainStoragePath("")
+        .weblogicDomainStorageReclaimPolicy(STORAGE_RECLAIM_POLICY_RETAIN)
```

Comment:
This line can be removed.  This section is responsible for setting non-default values, and this property defaults to retain.

Reply:
Hmmm, I got the impression that this test (defaultInputsFile_hasCorrectContents) is for default values. For example:
        .loadBalancer(LOAD_BALANCER_TRAEFIK)
        .weblogicDomainStorageSize("10Gi")
        .weblogicDomainStorageType(STORAGE_TYPE_HOST_PATH)
        .startupControl(STARTUP_CONTROL_AUTO)

Reply:
You're right - I was confusing this with CreateDomainInputs.newInputs